File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hm = {}
        for i in range(len(nums)):
            if target - nums[i] not in hm:
                hm[nums[i]] = i
            else:
                return[hm[target - nums[i]], i]
            

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        HashMap<Integer, Integer> hm = new HashMap<>();

        int[] res = new int[2];
        for(int i=0;i<nums.length;i++){
            int diff = target - nums[i];
            if (!(hm.containsKey(diff))){
                hm.put(nums[i], i );
            }
            else{
                res[0] = hm.get(diff);
                res[1] = i;
            }

        }
            return res;


    }
}